WoWPro_Leveling/Wrath/Horde/WOTLK_Grizzly_Hills.lua

```diff
@@ -1,7 +1,7 @@
 local guide = WoWPro:RegisterGuide("JamGri7577", "Leveling", "Grizzly Hills", "WowPro Team", "Horde", 3)
 WoWPro:GuideNickname(guide, "Grizzly Hills")
 WoWPro:GuideName(guide, "Grizzly Hills")
-WoWPro:GuideNextGuide(guide, "ScoZul7778")
+WoWPro:GuideNextGuide(guide, "Zul'Drak")
```

Comment:
Which reminds me, I should start reporting bad next guide links.

Reply:
This link works. I have tested it a few times.